datasets/stroomstoringen/stroomstoringen.json

```diff
@@ -0,0 +1,148 @@
+{
+  "type": "dataset",
+  "id": "stroomstoringen",
+  "title": "Stroomstoringen",
+  "description": "Stroomstoringen (van vandaag) Liander netwerk regio Amsterdam.",
+  "dateCreated": "2021-06-14T00:00:00",
+  "dateModified": "2021-06-14T00:00:00",
+  "license": "Creative Commons, Naamsvermelding",
+  "homepage": "https://data.amsterdam.nl",
+  "language": "nl",
+  "status": "beschikbaar",
+  "version": "1.0.0",
+  "temporalUnit": "nvt",
+  "contactPoint": {
+    "name": "",
+    "email": "datadiensten.ois@amsterdam.nl"
+  },
+  "accrualPeriodicity": "dagelijks",
```

Comment:
Storingen klinkt als ongeplande incidenten. Zoiets zou toch veel vaker dan een keer per dag ververst moeten worden?

Reply:
Goeie, ja klopt, staat ook zo in Airflow, pas het hierop aan